roles/nova-control/tasks/novnc.yml

```diff
@@ -19,9 +19,9 @@
   unarchive: src=/opt/stack/novnc/novnc-0.5.1.tgz dest=/opt/stack/novnc/ copy=no
   when: nova.novnc_method == 'file'
 
-- name: ensure novnc dir exists
-  file: src=/opt/stack/novnc/noVNC-0.5.1 dest=/usr/share/novnc state=link
-  when: nova.novnc_method == 'file'
+#- name: ensure novnc dir exists
+#  file: src=/opt/stack/novnc/noVNC-0.5.1 dest=/usr/share/novnc state=link
+#  when: nova.novnc_method == 'file'
```

Comment:
because it's an erroneous task.   above it we create the directory in the unarchive and then its trying to set it as a symlink.